_includes/components/header-and-footer/Search/index.jsx

```diff
@@ -55,18 +55,6 @@ export default function SearchMarkup(props) {
               }}
             />
           </div>
-
-          <ResultsArea
```

Comment:
Can the implementation of ResultsArea be removed now?

Reply:
Just to confirm, ResultsArea is the suggested dropdown that shows as you are typing. If I remember correctly the consensus was that we will remove it for now.